src/modules/users/users.service.ts

```
@@ -1065,29 +1065,116 @@ export class UserService {
     });
   }
 
+  /**
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)